src/aap_eda/services/activation/engine/podman.py

```diff
@@ -153,17 +153,22 @@ def start(self, request: ContainerRequest, log_handler: LogHandler) -> str:
             log_handler.write(error_message, flush=True)
             raise exceptions.ContainerStartError(error_message) from e
 
-    def get_status(self, container_id: str) -> ActivationStatus:
+    def get_status(self, container_id: str) -> tuple[ActivationStatus, str]:
         if self.client.containers.exists(container_id):
             container = self.client.containers.get(container_id)
             if container.status == "exited":
                 exit_code = container.attrs.get("State").get("ExitCode")
+                error = container.attrs.get("State").get("Error", "")
                 if exit_code == 0:
-                    return ActivationStatus.COMPLETED
+                    return ActivationStatus.COMPLETED, ""
```

Comment:
@hsong-rh maybe have a success message across both k8s and podman, we could use that constant here